### DISCUSSION TITLE
prayer filter not working.

### DISCUSSION THREAD

**Discussion by MainMan11** [runelite/runelite#16685] [Q&A]
is anyone else having issues with prayer filters not working? i change it and nothing happens. also killed hespori earlier and wouldn't give me option to clear him.

**Comment by Ahnii-Dev** [comment]
I have the same problem with prayers. The interface doesn't change the layout at all, and I can confirm it's working on native mobile client (non Runelite).
Edit:
I submitted the prayer issue
"Your" Hespori bug isn't (as far as I know) documented jet, you could start a new issue documenting it

**Reply by YvesW** [reply]
Prayer filtering when the prayer plugin is enabled, is fixed in 71ebd6882d7711933b11329aa3f4eda1181a3d16
You can also hide prayers via reordering, so I recommend using that

**Reply by Ahnii-Dev** [reply]
Thank you, I was just made aware of the existing fix (after posting it as an issue).
I don't really know what you mean by "hide prayers via reordering"? Is that a future feature, or did I miss something?

**Reply by YvesW** [reply]
https://cdn.runescape.com/assets/img/external/oldschool/2023/newsposts/2023-06-14/PrayerWIP.mp4
